ui/components/multichain/token-list-item/price/percentage-change/percentage-change.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Box, Text } from '../../../../component-library';
+import {
+  Display,
+  FontWeight,
+  TextColor,
+  TextVariant,
+} from '../../../../../helpers/constants/design-system';
+
+const isValidAmount = (amount: number | null | undefined): boolean =>
```

Comment:
This same function is in multiple files ; can we add a helper file and import from that one place?